js/components/message.jsx

```diff
@@ -1,12 +1,16 @@
 var React = require('react');
+require('../../stylesheets/message.scss');
 
 var Message = React.createClass({
   render: function() {
+    var message = this.props.message;
     return (
-      <div className='message'>
-        [{this.props.message.timestamp}]
-        {this.props.message.user}:
-        {this.props.message.text}
+      <div className="message">
+        <div className="sender-stamp">
+          <div className="message-sender">{message.user}</div>
+          <div className="message-time">{this.props.displayTime}</div>
```

Comment:
why not use the TimePresenter here instead of passing displayTime down as a prop? i feel like "hey message component, you take this message data and figure out how to display yourself" makes sense.. also cuts down on things humans need to remember or look up (the name of the `displayTime` prop)